modules/apps/object/object-api/src/main/java/com/liferay/object/definition/util/ObjectDefinitionUtil.java

```diff
@@ -113,7 +113,8 @@ private static String _getInvokerFileNameRegex(
 		"com.liferay.frontend.data.set.impl",
 		"com.liferay.headless.builder.impl", "com.liferay.list.type.service",
 		"com.liferay.mcp.server", "com.liferay.notification.service",
-		"com.liferay.object.service", "com.liferay.site.initializer.cms"
+		"com.liferay.object.service", "com.liferay.site.initializer.cms",
+		"com.liferay.site.initializer.extender"
```

Comment:
@MrJohn1911 we have to apply what @joseabelenda applied [here](https://github.com/MarinhoFeliphe/liferay-portal/pull/417/files#diff-727f0ee9b8a925b323c668dc7050be9dba5abc2af25efebb3ebab337fbcaf930R1272) in this pull request:

https://github.com/MarinhoFeliphe/liferay-portal/pull/417

otherwise every site initialize (including the client extension based ones) will be able to change the system objects.

probably the best approach would be finish the work of Abelenda and put our change on top of it, let's discuss about this later.